deepmd/dpmodel/fitting/property_fitting.py

```diff
@@ -89,9 +88,9 @@ def __init__(
     ) -> None:
         self.task_dim = task_dim
         self.intensive = intensive
-        self.bias_method = bias_method
+        self.property_name = property_name
```

Comment:
it seems that you do not need to maintain the property_name anymore.  one may access it by var_name of the parent class.

Reply:
I have delete attribute `property_name` and rename all related variables.